lib/src/controller.dart

```diff
@@ -1270,6 +1270,12 @@ class MaplibreMapController extends ChangeNotifier {
     return _mapboxGlPlatform.getLayerIds();
   }
 
+  Future<List<String>> getSourceIds() async {
```

Comment:
Could you please add a short doc comment here, including the fact that it's not implemented on web, yet?

Reply:
yup, no problems